packages/office-ui-fabric-react/src/common/_themeOverrides.scss

```diff
@@ -9,22 +9,21 @@ $ms-color-themeLighter: "[theme:themeLighter, default: #deecf9]";
 $ms-color-themeLighterAlt: "[theme:themeLighterAlt, default: #eff6fc]";
 $ms-color-black: "[theme:black, default: #000000]";
 $ms-color-blackTranslucent40: "[theme:blackTranslucent40, default: rgba(0,0,0,.4)]";
-$ms-color-neutralDark: "[theme:neutralDark, default: #212121]";
-$ms-color-neutralPrimary: "[theme:neutralPrimary, default: #333333]";
-$ms-color-neutralPrimaryAlt: "[theme:neutralPrimaryAlt, default: #3c3c3c]";
-$ms-color-neutralSecondary: "[theme:neutralSecondary, default: #666666]";
-$ms-color-neutralSecondaryAlt: "[theme:neutralSecondaryAlt, default: #767676]";
-$ms-color-neutralTertiary: "[theme:neutralTertiary, default: #a6a6a6]";
-$ms-color-neutralTertiaryAlt: "[theme:neutralTertiaryAlt, default: #c8c8c8]";
-$ms-color-neutralQuaternary: "[theme:neutralQuaternary, default: #d0d0d0]";
-$ms-color-neutralQuaternaryAlt: "[theme:neutralQuaternaryAlt, default: #dadada]";
-$ms-color-neutralLight: "[theme:neutralLight, default: #eaeaea]";
-$ms-color-neutralLighter: "[theme:neutralLighter, default: #f4f4f4]";
-$ms-color-neutralLighterAlt: "[theme:neutralLighterAlt, default: #f8f8f8]";
+$ms-color-neutralDark: "[theme:neutralDark, default: #201f1e]";
+$ms-color-neutralPrimary: "[theme:neutralPrimary, default: #323130]";
+$ms-color-neutralPrimaryAlt: "[theme:neutralPrimaryAlt, default: #3b3a39]";
+$ms-color-neutralSecondary: "[theme:neutralSecondary, default: #605e5c]";
+$ms-color-neutralSecondaryAlt: "[theme:neutralSecondaryAlt, default: #8a8886]";
+$ms-color-neutralTertiary: "[theme:neutralTertiary, default: #a19f9d]";
+$ms-color-neutralTertiaryAlt: "[theme:neutralTertiaryAlt, default: #c8c6c4]";
+$ms-color-neutralQuaternary: "[theme:neutralQuaternary, default: #d2d0ce]";
+$ms-color-neutralQuaternaryAlt: "[theme:neutralQuaternaryAlt, default: #e1dfdd]";
+$ms-color-neutralLight: "[theme:neutralLight, default: #edebe9]";
+$ms-color-neutralLighter: "[theme:neutralLighter, default: #f3f2f1]";
+$ms-color-neutralLighterAlt: "[theme:neutralLighterAlt, default: #faf9f8]";
 $ms-color-accent: "[theme:accent, default: #0078d4]";
 $ms-color-white: "[theme:white, default: #ffffff]";
 $ms-color-whiteTranslucent40: "[theme:whiteTranslucent40, default: rgba(255,255,255,.4)]";
-$ms-color-yellowDark: "[theme:yellowDark, default: #d29200]";
```

Comment:
Looks like this might still be referenced:
> src/components/FolderCover/FolderCover.scss: Error: Undefined variable: "$ms-color-yellowDark".

Reply:
Hmmm... this is weird because these files were generated automatically by a script. @phkuo in your #8143 the script is doing the same thing and removes the `yellowDark` color but it's still referenced in FolderCover.scss in experiments package. Do you know what should I do with it? Just manually add it back?

@natalieethell you had to deal with the same issue I believe in one of your PRs not long time ago. Did you come up with a solution?

Reply:
Ah! I just noticed that Philip's PR is also removing this line. It looks like `yellowDark` was a new color added in this PR https://github.com/OfficeDev/office-ui-fabric-react/pull/7976 to the _themeOverrides file, but not to IPalette, which is how _themeOverrides.scss is generated from the script. We should consider adding `yellowDark` to IPalette.ts and DefaultPalette.ts

Reply:
Perfect, thanks! Philip added `yellowDark` to IPalette and DefaultPalette in his PR